.github/workflows/ci.yml

```diff
@@ -53,6 +53,57 @@ jobs:
           docker push ${{ env.DOCKER_REGISTRY_URL }}/ucentralgw:$tag
         done
 
+  docker-compose:
+    runs-on: ubuntu-20.04
+    needs: docker
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v2
+
+    - name: Create certificate and key files from Github secrets
+      env:
+        DIGICERT_TIP_WILDCARD_CERT: ${{ secrets.DIGICERT_TIP_WILDCARD_CERT }}
+        DIGICERT_TIP_WILDCARD_KEY: ${{ secrets.DIGICERT_TIP_WILDCARD_KEY }}
+      run: |
+        cat certificates/issuer.pem certificates/root.pem > docker-compose/ucentral-data/certs/restapi-ca.pem
+        echo "$DIGICERT_TIP_WILDCARD_CERT" > docker-compose/ucentral-data/certs/websocket-cert.pem
+        echo "$DIGICERT_TIP_WILDCARD_CERT" > docker-compose/ucentral-data/certs/restapi-cert.pem
+        echo "$DIGICERT_TIP_WILDCARD_KEY" > docker-compose/ucentral-data/certs/websocket-key.pem
+        echo "$DIGICERT_TIP_WILDCARD_KEY" > docker-compose/ucentral-data/certs/restapi-key.pem
+
+    - name: Instantiate Docker Compose deployment
+      env:
+        UCENTRALGW_TAG: ${{ github.sha }}
+      run: |
+        docker-compose -f docker-compose/docker-compose.yml --project-directory docker-compose/ up -d
```

Comment:
Just setting the working-directory to docker-compose and executing `docker-compose up -d` would be a bit simpler to read.